{{cookiecutter.repo_name}}/vite.config.js

```diff
@@ -4,23 +4,29 @@ import { defineConfig } from 'vite';
 
 
 export default defineConfig({
+  // Where the project's files are
+  root: resolve('./{{cookiecutter.repo_name}}/static_source/'),
+  // The public path where assets are served, both in development and in production.
   base: "/static/",
-  root: resolve('./frontend/'),
   resolve:{
     alias:{
-      '@' : resolve('./frontend')
+      // Use '@' in urls as a shortcut for './static_source'. (Currently used in CSS files.)
+      '@' : resolve('./{{cookiecutter.repo_name}}/static_source')
     },
   },
   build: {
     manifest: true, // adds a manifest.json
     rollupOptions: {
       input: {
-        main: resolve(__dirname, './frontend/js/main.ts'),
-        base: resolve(__dirname, './frontend/css/base.js'),
-        raw_tailwind: resolve(__dirname, './frontend/css/tailwind.js'),
+        /* The bundle's entry point(s).  If you provide an array of entry points or an object mapping names to 
+        entry points, they will be bundled to separate output chunks. */
+        components: resolve(__dirname, './{{cookiecutter.repo_name}}/static_source/js/components.ts'),
+        main: resolve(__dirname, './{{cookiecutter.repo_name}}/static_source/js/main.ts'),
+        styles: resolve(__dirname, './{{cookiecutter.repo_name}}/static_source/css/styles.js'),
+        raw_tailwind: resolve(__dirname, './{{cookiecutter.repo_name}}/static_source/css/tailwind.js'),
       }
     },
-    outDir:  '../{{cookiecutter.repo_name}}/static_source/', // puts the manifest.json in PROJECT_ROOT/static_source/
+    outDir:  './', // puts the manifest.json in PROJECT_ROOT/static_source/
```

Comment:
would be nice to update the comment, put manifest in static source for django to collect